src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -1812,8 +1812,11 @@ public ReplaceStatus ReplaceIfNeeded(bool batchCompleted, bool didWork)
 
                     if (batchCompleted)
                     {
+                        var currentTotalSizeOfJournalsInBytes = _environment.Journal.Files.Select(i => 4 * 1024 * (i.Available4Kbs + i.WritePosIn4KbPosition)).Sum();
```

Comment:
better to expose in JournalFile the Size using the IJournalWriter.NumberOfAllocated4Kb IMO?
Also this should at the end be of type Size, so you can easily compare to the MinimumTotalSizeOfJournalsToRunFlushAndSyncWhenReplacingSideBySideIndex

Reply:
Changed :)